### DISCUSSION TITLE
Does bootstrap provide any custom styling for HTML5 Video?

### DISCUSSION THREAD

**Discussion by anoopkumarseth** [twbs/bootstrap#36685] [Ideas] [answered]
I was trying to customize but couldn't make any better design

![image](https://user-images.githubusercontent.com/30388576/177612647-477a6449-8b68-480c-aee8-fee7b42a2ada.png)

**Comment by mdo** [comment]
No, there's no video styling. Just aspect ratio helpers for responsive sizing.

**Reply by anoopkumarseth** [reply]
aspect ratio helpers work for iframe and image layout shift but video styling for a website that matches the theme is also important so will you add this in the future?

I have checked using the pseudo-selectors which is possible as you did that in form-control for choose files. I did that by enabling shadow dom.

![image](https://user-images.githubusercontent.com/30388576/177689616-7be15dd0-d1a1-4d38-94ac-7378e0ef081e.png)

maybe it should be it helper section so that anyone can copy CSS and add to the project when needed

![image](https://user-images.githubusercontent.com/30388576/177689923-156f42fa-94c3-4f1e-9b2d-b9743b2182f9.png)

**Comment by vanillajonathan** [comment, marked as answer]
No, the styling of the video element is done by the web browser not by CSS style sheet. If you want to style the video player you need a HTML5 video player JavaScript library such as [Video.js](https://videojs.com).
